### PR TITLE
Add chapter "What is NEAT"

### PR DESCRIPTION
**Pull request by jeremystucki**

### CODE REVIEW

**Review by janhohenheim — APPROVED** [review]

File: sections/what is neat.tex [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NEAT stands for Evolving Neural Networks through Augmenting Topologies and is a technology first proposed by O. Stanley. \cite{Stanley2002} \\
+It proposes an elegant way to combine genetic algorithms with evolving topologies.
```
Comment (janhohenheim) [inline sections/what is neat.tex:2]: What is a topology?

File: sections/what is neat/topology.tex [hunk 2]
```diff
@@ -0,0 +1,5 @@
+In traditional neural networks, the topology is fixed. The number of hidden layers and the number of neurons in each hidden layer are given. This makes it very easy to see the difference between two networks, since the only differences are the weights.
+
+The downside is, that the performance of these networks heavily depends on the chosen topology, which leads to the conclusion that many networks would perform better if one had chosen a different topology.
+
+NEAT proposes a technique to evolve the topology over time which allows the network to be better structured for a specific task then a configuration with hyper parameters.
```
Comment (janhohenheim) [inline sections/what is neat/topology.tex:5]: How?

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: sections/what is neat.tex (hunk 1 above)
Comment (jeremystucki) [reply sections/what is neat.tex:2]: I think we should add this to the chapter "Introduction to neural networks"

File: sections/what is neat/topology.tex (hunk 2 above)
Comment (jeremystucki) [reply sections/what is neat/topology.tex:5]: Explained in the "Speciation" subsection

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**